content/_changelogs/11.0.1.md

```diff
@@ -0,0 +1,19 @@
+## 11.0.1
+
+_Released 11/09/2022_
+
+**Bugfixes:**
+
+- Fixed an issue that caused enabling `experimentalSessionAndOrigin` to throw a
```

Comment:
Can we call out that these are fixing regressions in 11.0?

Reply:
Done